apps/frontend/scripts/getAvailableIcons.ts

```diff
@@ -0,0 +1,27 @@
+import fs from 'fs';
+function pascalToKebab(s: string) {
```

Comment:
I don't understand the problem, but I can't import package modules. Maybe it's because I'm using Node (which isn't really an explanation). I'll put a TODO in front of it.